Source/Scene/Cesium3DTileset.js

```diff
@@ -309,6 +320,31 @@ define([
         });
     };
 
+    function loadRefiningTiles(refiningTiles) {
```

Comment:
Would it be a good idea to call this function something else than `load...` to make it sound less like having to do with loading tile content? `prepareRefiningTiles`, `trackDescendantsWithContent` or something along those lines?